R/plot_shinygosling.R

```diff
@@ -84,11 +88,61 @@ plot_granges <-
         genomicFields = c("start", "end")
     )
 
-    ## Define tracks
+    ## This fixes the bug if .gosling directory does not already exist
+    cache_dir <- file.path(tools::R_user_dir("AlphaMissenseR", which = "cache"), ".gosling")
+    if (!dir.exists(cache_dir))
+        ## TODO: check return value to ensure directory is created successfully
+        dir.create(cache_dir, recursive = TRUE)
+    
+    ## trigger the option for bars or lolipop        
+    if (plot_type =="bars"){
```

Comment:
As best practice use 'identical(plot_type, "bars")`, since this returns TRUE or FALSE, whereas as written this can return cryptic errors if, e.g., `plot_type` where NA or "" (I realize that the checks above ensure that this is not the case, but that does not mean that the better practice of using `identical()` should be avoided).

Reply:
c9e9088